### DISCUSSION TITLE
Extending how to use http proxies？

### DISCUSSION THREAD

**Discussion by Brantfang** [microsoft/vscode-discussions#1120] [Extension Development QnA] [answered]
I want to develop an extension that uses axios as a web request tool. But I found that after configuring the axios proxy I can't request through the proxy server.
```
const instance = axios.create({
  httpsAgent: new HttpsProxyAgent.HttpsProxyAgent(`http://${proxy.host}:${proxy.port}`),
  proxy: false
});
```

**Comment by isidorn** [comment, marked as answer]
Sounds like out of scope for vscode extension api to me

**Reply by Brantfang** [reply]
I just want to know if the extension is not working after setting the proxy and if the proxy is blocked by the vscode extension host.
